kibbeh/src/lib/showErrorToast.ts

```diff
@@ -1,6 +1,7 @@
 import { useErrorToastStore } from "../modules/errors/useErrorToastStore";
+import { ToastDurations } from "../ui/Toast";
 
-export const showErrorToast = (m: string) => {
+export const showErrorToast = (m: string, duration?: ToastDurations, button?: JSX.Element, onClose?: () => void) => {
   console.log("showErrorToast: ", m);
-  useErrorToastStore.getState().showToast({ message: m });
+  useErrorToastStore.getState().showErrorToast({ message: m, duration, button, onClose });
```

Comment:
why are you changing toasts for deafen?

Reply:
don't know why this happened, reverted it